cmd/server/main.go

```diff
@@ -148,15 +149,20 @@ func addPingRoute(r *mux.Router) {
 	})
 }
 
-func setupSearcher(logger log.Logger, s *searcher, achPath, wirePath string) error {
-	logger.Logf("search: loading %s for ACH data", achPath)
-	if err := s.readFEDACHData(achPath); err != nil {
-		return fmt.Errorf("error reading ACH file at %s: %v", achPath, err)
+func setupSearcher(logger log.Logger, s *searcher, achFile, wireFile *os.File) error {
+	if achFile == nil || wireFile == nil {
+		return errors.New("missing fedach or fedwire data file(s)")
```

Comment:
I recommend breaking these into separate if-statements for a better error msg to the user.

Reply:
Yep, good call.